.eslintrc.js

```diff
@@ -14,6 +14,13 @@ module.exports = {
   },
   plugins: ['react'],
   rules: {
-    'react/react-in-jsx-scope': 'off'
+    'react/prop-types': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'prettier/prettier': [
+      'error',
+      {
+        endOfLine: 'auto'
+      }
+    ]
```

Comment:
Yea these are added to the `.prettierrc` already so you can remove this from here